### PR TITLE
Add shape function for squeeze.dims op

### PR DESCRIPTION
**Pull request by vivekkhandelwal1**
Changes to `_native_batch_norm_legit` and `upsample_nearest2d` in `serialized_shape_function_registry.cpp` are made just because this file is auto-generated, and the file was not auto-generated after the changes in `_shape_functions.py` for those two ops.

Signed-Off By: Vivek Khandelwal <vivek@nod-labs.com>

### CODE REVIEW

**Comment by vivekkhandelwal1** [comment]
Hi @davidberard98, could you please review this PR?

File: torch/jit/_shape_functions.py [hunk 1]
```diff
@@ -426,6 +426,14 @@ def squeeze(li: List[int], dim: int):
...
     return out
 
+def squeeze_dims(li: List[int], dims: List[int]):
+    wrapped_dims : List[int] = []
+    for i in dims:
+        wrapped_dims.append(maybe_wrap_dim(i, len(li)))
+    wrapped_dims.sort(reverse=True)
+    for i in wrapped_dims:
+        li = squeeze(li, i)
+    return li
```
Comment (davidberard98) [inline torch/jit/_shape_functions.py:436]: do you need add_shape_compute_mapping?

(also, can you comment in the PR clarifying that the _native_batch_norm_legit and upsample_nearest2d changes are unrelated & only occur because gen_jit_shape_functions.py wasn't run when the changes affecting the codegen changed?)

**Comment by vivekkhandelwal1** [comment]
Hi @davidberard98, I have addressed the issues. Could you please merge this PR now? Sorry, for addressing the issues late.

**Comment by vivekkhandelwal1** [comment]
Hi @jansel, could you please review this PR?

**Comment by vivekkhandelwal1** [comment]
@Chillee Could you please review/merge this PR?

**Review by davidberard98 — APPROVED** [review]
lgtm

**Comment by vivekkhandelwal1** [comment]
> lgtm

Hi @davidberard98, thanks for approving the PR. Could you please merge it for me since I don't have merge access?
